dependencies/README.md

```diff
@@ -0,0 +1,62 @@
+Static Linkage Checker
+======================
+
+Static Linkage Checker is a tool that inspects classpaths for [static linkage check](
+../library-best-practices/glossary.md#static-linkage-check).
+The tool identifies static linkage errors that appear in Java projects at runtime.
```

Comment:
This makes it sound like the Static Linkage Checker executes the code. I feel like the information in this sentence is already implied by "static linkage check" and can be removed. Maybe you can add the following to the prior sentence: "... and reports static linkage errors to the console."